java/flight/flight-jdbc-driver/src/main/java/org/apache/arrow/driver/jdbc/accessor/ArrowFlightJdbcAccessor.java

```diff
@@ -128,18 +128,30 @@ public BigDecimal getBigDecimal(final int i) {
   }
 
   @Override
-  public InputStream getAsciiStream() {
-    throw getOperationNotSupported(this.getClass());
+  public InputStream getAsciiStream() throws SQLException {
+    try {
+      throw getOperationNotSupported(this.getClass());
```

Comment:
You shouldn't `throw` and `catch` right after... I suggest you make `getOperationNotSupported` return a SQLException and just throw it like before

Reply:
Done.